templates/component_template.js

```diff
@@ -0,0 +1,13 @@
+import React, { Component } from 'react';
```

Comment:
I personally feel that this template is too generic. As you can see, we are not importing any of the grommet components here.

I like the idea of having these types of generators but it would be great if we could stay away from maintaining such structure. 

We recently created a project called `grommet-templates` where we aim to add more high level components structure that people usually spend a lot of time creating.

For the base react component structure there are few tools available:

For Atom: https://atom.io/packages/react-snippets
For Sublime: https://github.com/babel/babel-sublime-snippets

These tools are "IDE" specific. 

Your seems like IDE-agnostic which is great, but I feel that it should live outside the grommet cli tool.

@ericsoderberghp I would appreciate your feedback here as well.